### ISSUE TITLE
Get Info for Subscription does not work on 10.8 or 10.9

### ISSUE THREAD

**Issue by josh64x2** [ViennaRSS/vienna-rss#862]
First reported over at the forums: https://forums.cocoaforge.com/viewtopic.php?f=18&t=26957
Seems to be related to `SNDisclosableView` and related classes...

Console messages (from 10.8.5):
```
15/04/17 3:01:28.947 PM	Vienna[224]	Unknown class SNDisclosureButton in Interface Builder file at path /Users/josh/Desktop/Vienna.app/Contents/Resources/en.lproj/InfoWindow.nib.
15/04/17 3:01:28.949 PM	Vienna[224]	Unknown class SNDisclosureButton in Interface Builder file at path /Users/josh/Desktop/Vienna.app/Contents/Resources/en.lproj/InfoWindow.nib.
15/04/17 3:01:28.950 PM	Vienna[224]	Unknown class SNDisclosureButton in Interface Builder file at path /Users/josh/Desktop/Vienna.app/Contents/Resources/en.lproj/InfoWindow.nib.
15/04/17 3:01:28.953 PM	Vienna[224]	Unknown class SNDisclosureButton in Interface Builder file at path /Users/josh/Desktop/Vienna.app/Contents/Resources/en.lproj/InfoWindow.nib.
15/04/17 3:01:28.956 PM	Vienna[224]	Unknown class 'SNDisclosableView', using 'NSView' instead. Encountered in Interface Builder file at path /Users/josh/Desktop/Vienna.app/Contents/Resources/en.lproj/InfoWindow.nib.
15/04/17 3:01:28.957 PM	Vienna[224]	Unknown class 'SNDisclosableView', using 'NSView' instead. Encountered in Interface Builder file at path /Users/josh/Desktop/Vienna.app/Contents/Resources/en.lproj/InfoWindow.nib.
15/04/17 3:01:28.958 PM	Vienna[224]	Unknown class 'SNDisclosableView', using 'NSView' instead. Encountered in Interface Builder file at path /Users/josh/Desktop/Vienna.app/Contents/Resources/en.lproj/InfoWindow.nib.
15/04/17 3:01:28.959 PM	Vienna[224]	Unknown class 'SNDisclosableView', using 'NSView' instead. Encountered in Interface Builder file at path /Users/josh/Desktop/Vienna.app/Contents/Resources/en.lproj/InfoWindow.nib.
15/04/17 3:01:28.961 PM	Vienna[224]	An uncaught exception was raised
15/04/17 3:01:28.963 PM	Vienna[224]	[<NSView 0x7f9e216b6ce0> setValue:forUndefinedKey:]: this class is not key value coding-compliant for the key isShown.
15/04/17 3:01:28.969 PM	Vienna[224]	(
	0   CoreFoundation                      0x00007fff82aacb06 __exceptionPreprocess + 198
	1   libobjc.A.dylib                     0x00007fff8c8143f0 objc_exception_throw + 43
	2   CoreFoundation                      0x00007fff82b40f49 -[NSException raise] + 9
	3   Foundation                          0x00007fff899f1145 -[NSObject(NSKeyValueCoding) setValue:forKey:] + 408
	4   Foundation                          0x00007fff89a5492f -[NSObject(NSKeyValueCoding) setValue:forKeyPath:] + 350
	5   AppKit                              0x00007fff80d627ca -[NSView setValue:forKeyPath:] + 392
	6   AppKit                              0x00007fff80d020c9 -[NSIBUserDefinedRuntimeAttributesConnector establishConnection] + 332
	7   AppKit                              0x00007fff80da709a -[NSIBObjectData nibInstantiateWithOwner:topLevelObjects:] + 1012
	8   AppKit                              0x00007fff80d8611d loadNib + 317
	9   AppKit                              0x00007fff80d8585c +[NSBundle(NSNibLoading) _loadNibFile:nameTable:withZone:ownerBundle:] + 750
	10  AppKit                              0x00007fff80ed858c +[NSBundle(NSNibLoading) loadNibFile:externalNameTable:withZone:] + 140
	11  AppKit                              0x00007fff80f0b5db -[NSWindowController loadWindow] + 199
	12  AppKit                              0x00007fff80f0b2b5 -[NSWindowController window] + 77
	13  AppKit                              0x00007fff80f4bb79 -[NSWindowController showWindow:] + 53
	14  Vienna                              0x0000000100fd8750 Vienna + 468816
	15  Vienna                              0x0000000100fad065 Vienna + 290917
	16  AppKit                              0x00007fff80ed6959 -[NSApplication sendAction:to:from:] + 342
	17  AppKit                              0x00007fff8100c36c -[NSMenuItem _corePerformAction] + 406
	18  AppKit                              0x00007fff8100c05a -[NSCarbonMenuImpl performActionWithHighlightingForItemAtIndex:] + 133
	19  AppKit                              0x00007fff80cf920f -[NSMenu _internalPerformActionForItemAtIndex:] + 36
	20  AppKit                              0x00007fff80cf9097 -[NSCarbonMenuImpl _carbonCommandProcessEvent:handlerCallRef:] + 135
	21  AppKit                              0x00007fff81005165 NSSLMMenuEventHandler + 342
	22  HIToolbox                           0x00007fff89206d1a _ZL23DispatchEventToHandlersP14EventTargetRecP14OpaqueEventRefP14HandlerCallRec + 1206
	23  HIToolbox                           0x00007fff892061e9 _ZL30SendEventToEventTargetInternalP14OpaqueEventRefP20OpaqueEventTargetRefP14HandlerCallRec + 410
	24  HIToolbox                           0x00007fff8921bfc9 SendEventToEventTarget + 40
	25  HIToolbox                           0x00007fff89252ca9 _ZL18SendHICommandEventjPK9HICommandjjhPKvP20OpaqueEventTargetRefS5_PP14OpaqueEventRef + 443
	26  HIToolbox                           0x00007fff891f7a21 SendMenuCommandWithContextAndModifiers + 59
	27  HIToolbox                           0x00007fff891f79d3 SendMenuItemSelectedEvent + 254
	28  HIToolbox                           0x00007fff891f785f _ZL19FinishMenuSelectionP13SelectionDataP10MenuResultS2_ + 94
	29  HIToolbox                           0x00007fff8936de79 _ZL19PopUpMenuSelectCoreP8MenuData5PointdS1_tjPK4RecttjS4_S4_PK10__CFStringPP13OpaqueMenuRefPt + 1673
	30  HIToolbox                           0x00007fff8936d794 _HandlePopUpMenuSelection7 + 629
	31  AppKit                              0x00007fff8108856b _NSSLMPopUpCarbonMenu3 + 3916
	32  AppKit                              0x00007fff810875f8 -[NSCarbonMenuImpl _popUpContextMenu:withEvent:forView:withFont:] + 189
	33  AppKit                              0x00007fff811e1ff3 -[NSMenu _popUpContextMenu:withEvent:forView:withFont:] + 200
	34  AppKit                              0x00007fff813a7937 -[NSView rightMouseDown:] + 130
	35  AppKit                              0x00007fff810d6cfb -[NSControl _rightMouseUpOrDown:] + 534
	36  AppKit                              0x00007fff80ecbb81 -[NSWindow sendEvent:] + 8504
	37  AppKit                              0x00007fff80ec7644 -[NSApplication sendEvent:] + 5761
	38  Vienna                              0x0000000100ff285d Vienna + 575581
	39  AppKit                              0x00007fff80ddd21a -[NSApplication run] + 636
	40  AppKit                              0x00007fff80d81bd6 NSApplicationMain + 869
	41  libdyld.dylib                       0x00007fff8abff7e1 start + 0
	42  ???                                 0x0000000000000002 0x0 + 2
)
```

**Comment by josh64x2** [comment]
@Eitot are you able to confirm on 10.9 if you still have it installed?

**Comment by Eitot** [comment]
I'll check later. Is this working in 3.1.8?

**Comment by josh64x2** [comment]
OK I installed 10.9 and the same problem with the same console logs.
It's working in 3.1.8 but not in 3.1.9.

**Comment by Eitot** [comment]
From the logs it looks as though there is a missing dependency. Maybe it has to do with the use of modules and automatic framework linking. Not sure why this affects Mavericks though and only in this particular case.

**Comment by Eitot** [comment]
This error is also logged in Xcode on Sierra. I will try building from a few older commits to trace the error.

**Comment by Eitot** [comment]
Found it: https://github.com/ViennaRSS/vienna-rss/commit/aeef70dcdf5bb65828641978a31a341ff2cbe29a#diff-e957fc051af3f61dcf1ab78a80472b93L1792.

I accidentally removed the manual link when I applied the commit to the legacy branch. The DisclosableView framework had already been removed on the master branch.

**Comment by josh64x2** [comment]
Oh great job! I did have a look at the modules in interface builder but without them being linked it didn't solve anything 😜

**Comment by Eitot** [comment]
Fixed in #863.
